Documentation/contributing/development/dev_setup.rst

```diff
@@ -66,8 +66,15 @@ setup for the Vagrantfile in the root of the Cilium tree depends on a number of
 environment variables and network setup that are managed via
 ``contrib/vagrant/start.sh``.
 
-Using the provided Vagrantfile
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Using the provided Vagrantfiles
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+
+   Vagrantfiles assume that the name of your Cilium directory is
+   ``cilium``. This allows mounting of the parent directory to the VMs,
```

Comment:
For the same reason as highlighted earlier, I have a bunch of Cilium repositories named `cilium`, `cilium2`, etc. Probably a corner case, but maybe we don't even need to enforce the `cilium` name if we setup a separate synced folder for the `clustermesh-apiserver` repository.